presto-jwt-authenticators/src/main/java/com/facebook/presto/jwt/PrestoJWTAuthenticatorFactory.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.jwt;
+
+import com.facebook.airlift.bootstrap.Bootstrap;
+import com.facebook.presto.spi.security.JWTAuthenticator;
+import com.facebook.presto.spi.security.JWTAuthenticatorFactory;
+import com.google.inject.Injector;
+import com.google.inject.Scopes;
+
+import java.util.Map;
+
+import static com.facebook.airlift.configuration.ConfigBinder.configBinder;
+import static com.google.common.base.Throwables.throwIfUnchecked;
+
+public class PrestoJWTAuthenticatorFactory
+        implements JWTAuthenticatorFactory
+{
+    @Override
+    public String getName()
+    {
+        return "default-jwt";
```

Comment:
nit: since we have renamed the class names, let's rename this to presto-jwt as well.